### DISCUSSION TITLE
Generate a sarif log file with only warnings and errors?

### DISCUSSION THREAD

**Discussion by calicoder77** [dotnet/msbuild#8609] [Q&A] [answered]
Good day, hope everyone is doing well. I got the following **msbuild.exe** command that successfully generates a log.sarif file for me from my project. Is there a way to only log in that sarif file the warnings and errors and not every artifact that gets checked?

`msbuild --% -t:Rebuild -p:RunCodeAnalysis=true -p:ClOptions="/analyze:log log.sarif%20/analyze:log:compilerwarnings" -p:Configuration=Release Project1.sln`

Appreciate the tips and help, thank you.

**Comment by rainersigwald** [comment, marked as answer]
That's a question for the C++ compiler's SARIF implementation, so I think you'll have better luck asking by [opening a feedback ticket instead](https://docs.microsoft.com/visualstudio/ide/feedback-options).

**Reply by calicoder77** [reply]
Thank you, I'll go move it over there
